modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
@baartosz Guzzle is a better choice as you don't need an abstraction since PSR7 is your preferred abstraction, no need for an abstraction over an abstraction.